src/main/java/tds/exam/results/configuration/ExamResultsTransmitterServiceProperties.java

```diff
@@ -13,7 +13,10 @@
     private String tisCallbackUrl = "";
     private boolean validateTrtXml = false;
     private boolean sendToTis = false;
-    private boolean retryOnError = true;
+    private long retryInitialInterval = 1000;
+    private double retryIntervalMultiplier = 2;
+    private long retryMaxInterval = 5000;
+    private int retryAmount = 3;
```

Comment:
We went off [this](http://docs.spring.io/autorepo/docs/spring-amqp-dist/1.5.6.RELEASE/api/org/springframework/amqp/rabbit/config/RetryInterceptorBuilder.html).  The way I read it is the max retry amounts is the number of times it'll try to retry a message before it calls it failed which the message can either be sent to a dead letter queue or logged (we're logging).  

Each time Spring retries the message it increases the interval by 2.  So for the first one it'll wait for a second then the next time 2 seconds and then the third time 4.  With the numbers here as you both have noted the max is never hit.

However, these are all configurable so let's say a deployment wants to retry 100 times before failing.  They leave our numbers.  If we go to our example above and keep growing our retry interval we'd hit the max (5 seconds) on the fourth attempt.  So for every attempt after you reach the max interval the message would be retried every five seconds with what we have configured.

@malaffoon Our specific use case is that if for some reason there is a network hiccup between this processor and TIS we want to retry.  However, at this point in time we do not guarantee that the message is sent to TIS since the 2016 didn't either.  It will log the message to Kibana which at that time a dev ops support person could find a way to have it reprocessed.